samples/drivers/gnss/src/main.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 Trackunit Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gnss.h>
+
+static void gnss_data_cb(const struct device *dev, const struct gnss_data *data)
+{
+	if (data->info.fix_status != GNSS_FIX_STATUS_NO_FIX) {
+		printk("%s has fix!\r\n", dev->name);
```

Comment:
nit: you can use printf and drop kernel.h include

Reply:
updated to use printf